tests/pos/i11114.scala

```diff
@@ -0,0 +1,4 @@
+class Foo {
+  val x: (Int, Int) = (1, 1)
+  val y: Int = x._1
+}
```

Comment:
Perhaps create a bytecode test in `DottyBytecodeTests.scala` with that snippet, to ensure that `x._1` is indeed compiled as `x._1$mpI$sp` or whatever it's called.